docs/multi-stage-query/reference.md

```diff
@@ -90,6 +93,93 @@ can precede the column list: `EXTEND (timestamp VARCHAR...)`.
 
 For more information, see [Read external data with EXTERN](concepts.md#read-external-data-with-extern).
 
+#### `EXTERN` to export to a destination
+
+`EXTERN` can be used to specify a destination, where the data needs to be exported.
+This variation of EXTERN requires one argument, the details of the destination as specified below.
+This variation additionally requires an `AS` clause to specify the format of the exported rows.
```

Comment:
```suggestion
This variation of EXTERN requires two arguments: the details of the destination and an `AS` clause to specify the format of the exported rows.
```

Reply:
The `AS` clause would not be an argument to extern, it's present elsewhere in the query

Reply:
How about the change I just made?